src/customer/RegularCustomer.java

```diff
@@ -0,0 +1,55 @@
+//user defined package
+package customer;
```

Comment:
specified package doesn't show a relationship with other packages of the same program modify to something like 
```suggestion
package com.debonair.customer;
```